docs/docs/workflows/settings/README.md

```diff
@@ -40,7 +40,15 @@ On error, the step will export a `$summary` property that tells you how many tim
 
 If the step execution succeeds during any retry, the execution will proceed to the next step of the workflow.
 
-If the step fails on all 8 retries, it throws the final error, and you should receive [an error notification](/workflows/errors/) through your standard notification channel.
+If the step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel.
+
+### Send error notifications on the first error
+
+By default, if a step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes you need to investigate errors as soon as they happen. If you're connecting to your database, and receive an error that the DB is down, you may want to investigate that immediately.
```

Comment:
```suggestion
By default, if a step fails on all 8 retries and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes, you need to investigate errors as soon as they happen. If you're connecting to your database and receive an error that the DB is down, you may want to investigate that immediately.
```

Reply:
Removed comma before 'and' in a compound predicate; Added comma after introductory phrase